From 63ed60cb35e2435773150972a5266ec470f7bdff Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 22 Mar 2024 17:26:12 +0800
Subject: [PATCH 04/13] fix(scripts): use ABI2 (New-World) Mihomo binary for
 loongarch64

Currently, only the alpha Mihomo binary distinguishes between abi1 (old-world)
and abi2 (new-world) binaries.
---
 scripts/check.mjs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/check.mjs b/scripts/check.mjs
index 19e2ac6..ccf6e89 100644
--- a/scripts/check.mjs
+++ b/scripts/check.mjs
@@ -68,7 +68,7 @@ const META_ALPHA_MAP = {
   "linux-arm64": "mihomo-linux-arm64",
   "linux-arm": "mihomo-linux-armv7",
   "linux-riscv64": "mihomo-linux-riscv64",
-  "linux-loong64": "mihomo-linux-loong64",
+  "linux-loong64": "mihomo-linux-loong64-abi2",
 };
 
 // Fetch the latest alpha release version from the version.txt file
-- 
2.45.2

